.gitignore

```diff
@@ -3,9 +3,12 @@
 # If you find yourself ignoring temporary files generated by your text editor
 # or operating system, you probably want to add a global ignore instead:
 #   git config --global core.excludesfile '~/.gitignore_global'
+# Ignore secrets
+.envrc
 
 # Ignore bundler config.
 /.bundle
+.
 
```

Comment:
@whereismyjetpack  do we need this one`/vendor/*` as well?

Reply:
I was going to address .gitignore as another PR since this one is getting super funk-a-tron. 

there's a few local paths. do you want me to add them in now?

Reply:
it is totally fine with me to add more to gitignore, but yeah I hear you for when PRs gets biggers :)

Reply:
ok. I've added them to this one. I also added some docs on how rspec runs. (this process may continue to evolve. please provide feedback)